### DISCUSSION TITLE
Only UsersSeeder is seeded

### DISCUSSION THREAD

**Discussion by vimkaf** [leafsphp/leaf#183] [General]
when running the command `php leaf db:seed` only the UsersSeeder is being seeded all other generated seeders are not running.

**Comment by vimkaf** [comment]
I have resolved this by adding by adding the seeders to the DatabaseSeeder file. I would recommend this be updated in the docs. I could not find anything relating to this.

**Reply by mychidarko** [reply]
Adding this in the next docs update, thanks
